<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-S2ELRE8" ignored="0" name="BankingApp Test" tests="6" failures="2" timestamp="05 Jan 2019 14:30:00 GMT" time="80.417" errors="0">
  <testcase name="LoginTest" time="5.61" classname="com.inetbanking.testCases.TC_LoginTest_001"/>
  <testcase name="loginDDT" time="13.727" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="7.288" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="12.794" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="6.785" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="12.377" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
</testsuite> <!-- BankingApp Test -->
